shared/js/background/background.js

```diff
@@ -22,8 +22,10 @@ import initReloader from './devbuild-reloader'
 // otherwise FF might miss the onInstalled event
 require('./events')
 const settings = require('./settings')
-require('./dnr-config-rulesets')
-require('./script-injection')
+if (BUILD_TARGET === 'chrome-mv3') {
+    require('./dnr-config-rulesets')
+    require('./script-injection')
```

Comment:
@sammacbeth is esbuild stripping these then?

Reply:
Yes. These modules will be removed from the build output.